libs/template/themes/src/storefront.theme.ts

```diff
@@ -1,18 +1,17 @@
 import { Theme } from '@spryker-oryx/experience';
+import {
+  storefrontIconSources,
+  storefrontMaterialIcons,
+} from '@spryker-oryx/presets/resources';
 import { defaultBreakpoints } from '@spryker-oryx/themes/breakpoints';
 
 export const storefrontTheme: Theme = {
   name: 'storefront',
   breakpoints: defaultBreakpoints,
   designTokens: () =>
     import('../design-tokens/src/storefront').then((s) => s.storefrontTokens),
-  // icons: {
-  //   resource: materialIcons,
-  //   resources: [
-  //     {
-  //       resource: fontAwesomeIcons,
-  //       types: [IconTypes.User],
-  //     },
-  //   ],
-  // },
+  icons: {
+    resource: storefrontMaterialIcons,
+    resources: storefrontIconSources,
```

Comment:
As suggested elsewhere in this PR we should endup with:

```ts
icons: {
    resource: materialDesignIcons,
}
```

Only if EB requires to keep storefrontIconSources, but for clearness we could rename it.